documentation/src/main/asciidoc/topics/con_cache_containers.adoc

```diff
@@ -1,16 +1,17 @@
+[id='cache_containers-{context}']
 = Cache Containers
-The {brandname} subsystem can declare multiple containers. A container is declared as follows:
+Cache containers declare one or more local or clustered caches that a Cache Manager controls.
 
+{brandname} subsystem declaration::
++
 [source,xml,options="nowrap",subs=attributes+]
 ----
 include::config_examples/ispn_subsystem_containers.xml[]
 ----
 
-Note that in server mode is the lack of an implicit default cache, but the ability to specify a named cache as the default.
-
-If you need to declare clustered caches (distributed, replicated, invalidation), you also need to specify the `<transport/>` element which references an existing JGroups transport. This is not needed if you only intend to have local caches only.
-
-[source,xml,options="nowrap",subs=attributes+]
-----
-include::config_examples/ispn_jgroups_transport.xml[]
-----
+//dnaro: need to clarify this note.
+[NOTE]
+====
+{brandname} servers let you specify named caches as default dynamically, rather
+than providing implicit default caches.
```

Comment:
The intent here was to contrast the server configuration, where the default cache is selected with the `<cache-container default-cache="<cache>">`, with the old embedded configuration, where we had a `<default>` element:

https://github.com/infinispan/infinispan/blob/6.0.x/core/src/test/resources/configs/batching.xml#L6

Embedded mode switched to the server style config in 7.0, so there's no point in mentioning the implicit default cache any more.